home/tests/test_content_import_export.py

```diff
@@ -13,12 +13,14 @@
 from django.core import serializers  # type: ignore
 from django.core.exceptions import ValidationError  # type: ignore
 from django.core.files.images import ImageFile  # type: ignore
+from django.test import override_settings  # type: ignore
```

Comment:
pytest-django has a `settings` fixture that we're already using in our own `tmp_media_path` fixture.